src/sentry/snuba/events.py

```diff
@@ -559,6 +559,23 @@ class Columns(Enum):
         issue_platform_name="contexts.value",
         alias="contexts.value",
     )
+    APP_IN_FOREGROUND = Column(
+        group_name="events.contexts[app.in_foreground]",
+        event_name="contexts[app.in_foreground]",
+        transaction_name="contexts[app.in_foreground]",
+        discover_name="contexts[app.in_foreground]",
+        issue_platform_name="contexts[app.in_foreground]",
+        alias="app.in_foreground",
+    )
+    # TODO: review this
+    THREAD_MAIN = Column(
+        group_name="events.threads.values.main",
+        event_name="threads.values.main",
+        transaction_name="threads.values.main",
+        discover_name="threads.values.main",
+        issue_platform_name="threads.values.main",
+        alias="thread.main",
+    )
```

Comment:
I don't know if this works, apparently `threads` are not mapped anywhere else, maybe this won't work?

Reply:
Will make another PR